modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/ManageUserAssociatedDataEntitiesMVCRenderCommand.java

```diff
@@ -77,18 +77,33 @@ public String render(
 		UADEntityAggregator uadEntityAggregator =
 			_uadRegistry.getUADEntityAggregator(uadRegistryKey);
 
+		int count = uadEntityAggregator.count(selUserId);
+
 		searchContainer.setResults(
 			uadEntityAggregator.getUADEntities(
 				selUserId, searchContainer.getStart(),
 				searchContainer.getEnd()));
-		searchContainer.setTotal(uadEntityAggregator.count(selUserId));
+
+		searchContainer.setTotal(count);
+
+		String entitySetName = uadEntityAggregator.getUADEntitySetName();
+
+		ManageUserAssociatedDataEntitiesDisplay
+			manageUserAssociatedDataEntitiesDisplay =
+				new ManageUserAssociatedDataEntitiesDisplay(
+					_uadRegistry.getUADEntityDisplay(uadRegistryKey),
+					searchContainer);
```

Comment:
In a case like this, Brian will probably want us to create the Display class first (we can do that if we're using a nullary constructor), then call each setter alphabetically by method name, creating each piece of data before the respective setter if necessary.